field_records/ubna_2023.csv

```diff
@@ -17,9 +17,15 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
-2023-06-16T11:19:00 , (DATE-TIME) , C , 004 , Foliage , 47° 39' 18.810'' N , 122° 17' 26.688'' W , 192000 , Medium , None , None , 4.119 , (VOLTAGE) , CT , YC , (UPLOADER) , (RECOVER-DATE) , Panasonic Batteries Nighttime 3:00-13:30 UTC Need longer cable Make Note of Moisture
-2023-06-16T11:19:00 , (DATE-TIME) , D , 006 , Foliage , 47° 39' 18.810'' N , 122° 17' 26.688'' W , 48000 , Medium , None , None , 4.124 , (VOLTAGE) , CT , YC , (UPLOADER) , (RECOVER-DATE) , Panasonic Batteries Daytime 0:00-24:00 UTC Need longer cable
-2023-06-16T11:39:00 , (DATE-TIME) , E , 012 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 192000 , Medium , None , None , 4.051 , (VOLTAGE) , CT , YC , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Nighttime 3:00-13:30 UTC
-2023-06-16T11:39:00 , (DATE-TIME) , F , 009 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 48000 , Medium , None , None , 4.072 , (VOLTAGE) , CT , YC , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC
\ No newline at end of file
+2023-06-16T10:57:00 , 2023-06-22T13:00:00 , A , 007 , Telephone Field , 47° 39' 29.520'' N , 122° 17' 26.310'' W , 48000 , Medium , None , None , 4.012 , 3.583 , CT , YC , MB , recover-20230622 , Ikea Batteries Daytime 0:00-24:00 UTC
+2023-06-16T10:57:00 , 2023-06-22T13:00:00 , B , 008 , Telephone Field , 47° 39' 29.520'' N , 122° 17' 26.310'' W , 192000 , Medium , None , None , 3.965 , 3.305 , CT , YC , MB , recover-20230622 , Ikea Batteries Nighttime 3:00-13:30 UTC
+2023-06-16T11:19:00 , 2023-06-22T13:19:00 , C , 004 , Foliage , 47° 39' 18.810'' N , 122° 17' 26.688'' W , 192000 , Medium , None , None , 4.119 , 3.782 , CT , YC , MB , recover-20230622 , Panasonic Batteries Nighttime 3:00-13:30 UTC Need longer cable Make Note of Moisture
+2023-06-16T11:19:00 , 2023-06-22T13:19:00 , D , 006 , Foliage , 47° 39' 18.810'' N , 122° 17' 26.688'' W , 48000 , Medium , None , None , 4.124 , 3.800 , CT , YC , MB , recover-20230622 , Panasonic Batteries Daytime 0:00-24:00 UTC Need longer cable
+2023-06-16T11:39:00 , 2023-06-22T13:41:00 , E , 012 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 192000 , Medium , None , None , 4.051 , 3.715 , CT , YC , MB , recover-20230622 , Ikea Batteries Nighttime 3:00-13:30 UTC
+2023-06-16T11:39:00 , 2023-06-22T13:41:00 , F , 009 , Central Pond , 47° 39' 17.502'' N , 122° 18' 3.060'' W , 48000 , Medium , None , None , 4.072 , 3.784 , CT , YC , MB , recover-20230622 , Ikea Batteries Daytime 0:00-24:00 UTC
+2023-06-22T13:07:00 , (DATE-TIME) , A , 001 , Telephone Field , 47° 39' 29.520'' N , 122° 17' 26.310'' W , 48000 , Medium , None , None , 4.140 , (VOLTAGE) , CT , MB , (UPLOADER) , (RECOVER-DATE) , Ikea Batteries Daytime 0:00-24:00 UTC Changed from daytime to nighttime might need to change previous values
```

Comment:
Will need to change this to the appropriate all-day configuration values.

Reply:
Remove the `Changed from daytime to nighttime might need to change previous values`. With the change above, this will be resolved.